docs/control-structures.rst

```diff
@@ -66,6 +66,28 @@ as the actual contract has not been created yet.
 Functions of other contracts have to be called externally. For an external call,
 all function arguments have to be copied to memory.
 
+.. warning::
+  Be careful that ``feed.info.value(10).gas(800)`` only locally sets the ``value`` and amount of ``gas`` sent with the function call, and the parentheses at the end perform the actual call. So in this case, the function is not called and the ``value`` and ``gas`` settings are lost.
```

Comment:
Please undo the move. `.value` is only introduced later on.

Reply:
Looks like this was fixed by https://github.com/ethereum/solidity/pull/6846 as in `develop` it's in the right place.